### PR TITLE
WIP Add CI/CD type to PUT method

### PR DESCRIPTION
**Pull request by guy-suli**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
Add the type for CI/CD.
The default value is still `cli`, in case it can be override by using orbs/github actions

### CODE REVIEW

File: checkov/common/bridgecrew/platform_integration.py [hunk 1]
```diff
@@ -16,7 +16,9 @@
 
 UNAUTHORIZED_MESSAGE = 'User is not authorized to access this resource with an explicit deny'
 
-BC_API_URL = os.getenv('BC_API_URL',"https://www.bridgecrew.cloud/api/v1")
+BC_API_URL = os.getenv('BC_API_URL', "https://www.bridgecrew.cloud/api/v1")
+BC_SOURCE = os.getenv('BC_SOURCE', "cli")
+BC_REPO = os.getenv('BC_REPO', "")
```
Comment (schosterbarak) [inline checkov/common/bridgecrew/platform_integration.py:21]: where do you use BC_REPO?

File: checkov/common/bridgecrew/platform_integration.py [hunk 2]
```diff
@@ -108,7 +110,7 @@ def commit_repository(self, branch):
         """
         request = None
         try:
-            request = http.request("PUT", INTEGRATIONS_API_URL,
+            request = http.request("PUT", f"{INTEGRATIONS_API_URL}/?source={BC_SOURCE}",
```
Comment (schosterbarak) [inline checkov/common/bridgecrew/platform_integration.py:113]: unittest?

File: checkov/common/bridgecrew/platform_integration.py (hunk 1 above)
Comment (guy-suli) [reply checkov/common/bridgecrew/platform_integration.py:21]: I will use it in the next phase, from which repo of orbs/github action it arrived.

File: checkov/common/bridgecrew/platform_integration.py (hunk 2 above)
Comment (guy-suli) [reply checkov/common/bridgecrew/platform_integration.py:113]: added

File: checkov/common/bridgecrew/platform_integration.py (hunk 1 above)
Comment (schosterbarak) [reply checkov/common/bridgecrew/platform_integration.py:21]: is organisation part of the repo name? i.e. bridgecrewio/checkov ?

Comment (guy-suli) [reply checkov/common/bridgecrew/platform_integration.py]: Deleted

**Pull request merged**